src/components/Footer/index.js

```diff
@@ -14,11 +13,11 @@ export const JoinSlack = () => {
 }
 
 
-const Address = styled.address`
+/*const Address = styled.address`
```

Comment:
Why is this needed?

Reply:
I've just commented the existing code. Can remove it in case we don't need it.

Reply:
pls delete this code